awx/conf/settings.py

```diff
@@ -186,8 +194,6 @@ def set_many(self, data, **kwargs):
             self.set(key, value, log=False, **kwargs)
 
     def _handle_encryption(self, method, key, value):
-        TransientSetting = namedtuple('TransientSetting', ['pk', 'value'])
```

Comment:
It turns out named tuples are *really slow to create on demand like this.  In my testing, this one line was accounting for almost 10% of the total time spent.

looks like other communities have noticed this about Python as well:

https://rhye.org/post/python-cassandra-namedtuple-performance/
https://bugs.python.org/issue28638